extension/js/background.js

```diff
@@ -176,62 +224,87 @@ function handleMessage(data, sender, sendRespones) {
         url: data.url,
     };
     //console.log(item)
-    db.put(item, function callback(err, result){
-        if(!err) {
+    db.put(item, function callback(err, result) {
+        if (!err) {
             console.log('Successfully stored the page: ' + data.url)
-        }
-        else {
+        } else {
             console.log('Error on URL: ' + data.url)
         }
 
     });
-}*/
+}
 
-/*function show_url(){
+function show_url() {
```

Comment:
I'm unsure of this function meaning, it seems a debug. It is supposed to show something, get something or it is only an intermediate step?